src/Compilers/VisualBasic/Test/Semantic/Semantics/Conversions.vb

```diff
@@ -605,7 +605,8 @@ End Class
                     New TypeAndValue(uint64Type, CULng(18)),
                     New TypeAndValue(decimalType, CDec(-11.3)),
                     New TypeAndValue(doubleType, CDbl(&HF000000000000000UL)),
-                    New TypeAndValue(doubleType, CDbl(&H70000000000000F0L)),
+                    New TypeAndValue(doubleType, CDbl(&H8000000000000000L)),
+                    New TypeAndValue(doubleType, CDbl(&H7FFFFFFFFFFFFC00L)),
```

Comment:
Should these cases be added to `ConstantExpressionConversions2` as well, at line 1169?